config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
-  resources :champions, only: [ :index, :new, :create ]
+  resources :champions, only: [ :index, :new, :create, :show ] do 
```

Comment:
it should be like this 
 resources :champions, only: [ :index, :new, :create ] do
    resources :bookings, only: [ :new, :create ]
  end

we need to somehow nest reviews also